data/na/swagger/1-0-5.yaml

```diff
@@ -2349,6 +2393,147 @@ definitions:
+        description: 'Tax included or not'
+      process_back_payments:
+        type: boolean
+        description: 'If a recurring payment is disabled and later reactivated back, payments will be charged if enabled.'
+  RecurringPaymentCard:
+    properties:
+      card_type:
+        type: string
+        maxLength: 2
+        description: 'The type of card used in the transaction. AM = American Express, DI = Diners, JB = JCB, MC = MasterCard, NN = Discover, VI = Visa, PV = Visa Debit, MD = Debit Mastercard'
+        enum: ['AM','DI','JB','MC','NN','VI','PV','MD']
+      name:
+        type: string
+        minLength: 1
+        maxLength: 64
+        description: Card holder name.
+      expiry_year:
+        type: string
+        maxLength: 2
+        description: eg. 25 for 2025.
+      expiry_month:
+        type: string
+        maxLength: 2
+        description: eg. 02 for February.
+  RecurringPaymentCustom:
+    properties:
+      order_number:
+        type: string
+        maxLength: 30
+        description: 'Order number'
+      comments:
+        type: string
+        maxLength: 2000
+        description: 'Comment regarding Recurring Payment account'
+      ref1:
+        type: string
+        maxLength: 256
+        description: 'custom reference field 1'
+      ref2:
+        type: string
+        maxLength: 256
+        description: 'custom reference field 2'
+      ref3:
+        type: string
+        maxLength: 256
+        description: 'custom reference field 3'
+      ref4:
+        type: string
+        maxLength: 256
+        description: 'custom reference field 4'
+      ref5:
+        type: string
+        maxLength: 256
+        description: 'custom reference field 5'
+  RecurringPaymentSchedule:
+    properties:
+      period:
+        type: string
+        description: 'The period is unit of measurement for the frequency of the payment: day, week, month or year.'
+      increment:
+        type: integer
+        description: 'The frequency of the transactions'
+      start_date:
+        type: string
+        description: 'Initial card charge date'
+      second_billing:
+        type: string
+        description: 'Card's second charge date'
+      expiry_date:
+        type: string
+        description: 'The last date of the billing cycle'
+      never_expires:
+        type: boolean
+        description: 'Card incurs charges with no end date'
+      bill_at_end_of_month:
+        type: boolean
+        description: 'Card will be charged at the end of the month'
+      upcoming_transactions:
+        type: array
+        maxLength: 3
+        items:
+          $ref: '#/definitions/UpcomingTransactionRecord'
+  UpcomingTransactionRecord:
+    properties:
+      billing_date:
+        type: string
+        description: 'Start date of the billing cycle'
+      billing_end:
+        type: string
+        description: 'Last date of the billing cycle'
+      amount:
+        type: number
+        format: double
```

Comment:
is there a format decimal for this and the others? double != decimal

Reply:
this amount as a property is used in other places, and thats how it is defined, as I can not test locally, I am just copying this.